cranelift/frontend/src/switch.rs

```diff
@@ -529,10 +531,12 @@ block3:
 block0:
     v0 = iconst.i8 0
     v1 = icmp_imm eq v0, 255  ; v0 = 0
-    brnz v1, block1
-    jump block4
+    brif v1, block1, block4
 
 block4:
+    jump block5
+
+block5:
```

Comment:
The TODO on line 523 is referring to this unnecessary indirection introduced by the switch compilation. I'm not sure if there's a good way to rework `build_search_branches` to avoid this, but perhaps it's not worth thinking too hard about as VCode will skip the intermeidate block.